core/spec.md

```diff
@@ -5528,6 +5566,35 @@ as defined below:
   - MUST NOT be present if the Resource's `hasdocument` model attribute is
     set to `false.
 
+##### `ancestor` Attribute
+- Type: String
+- Description: A string value that points to another version based on the
+  `versionid`-attribute.
+
+  The `ancestor` attribute MAY be set to any existing Version's `versionid`,
+  or to the `versionid` of the version itself, to indicate to the server that
+  this Version has no `ancestor`.
+
+  When creating a version without explicitly setting the `ancestor`
+  attribute, the server MUST set the `ancestor` to the most recent Version's
+  `versionid` attribute, based on the `createdat` attribute. If multiple
+  exist, the first one will be used in alphabetical order.
+
+  If an operation contains multiple versions with the `ancestor` attribute
+  omitted, the server MUST order all versions based on the `createdat`
+  attribute and then alphabetically. The first Version will have the most
+  recent Version's `versionid` as its `ancestor` as clarified above.
+
+  When deleting a Version, the server MUST update the `ancestor` attribute
+  of any Version that points to the deleted Version to point to itself.
+
+- Constraints: the `ancestor` attribute MUST NOT be set to a value that
+  creates circular references between versions. For example, an operation that
+  makes Version A's ancestor B, and version B's ancestor A, MUST be refused
+  by the server.
+  If the `ancestor` attribute is set to a `versionid` that does not exist
```

Comment:
Any attempt to set an `ancestor` attribute to a non-existing `versionid` MUST generate an error.